p2p/host/basic/basic_host.go

```diff
@@ -141,11 +145,11 @@ type HostOpts struct {
 }
 
 // NewHost constructs a new *BasicHost and activates it by attaching its stream and connection handlers to the given inet.Network.
-func NewHost(ctx context.Context, net network.Network, opts *HostOpts) (*BasicHost, error) {
+func NewHost(ctx context.Context, network network.Network, opts *HostOpts) (*BasicHost, error) {
```

Comment:
having the variable overload the import name so you can't refer to `network` anymore seems like it can add to confusion. `net` isn't great either, but ideally something non-overloaded?

Reply:
This is done.

Reply:
This is done.